### DISCUSSION TITLE
驱动分组初始化之后，点位信息从哪里加载的呢？

### DISCUSSION THREAD

**Discussion by MrColinWei** [emqx/neuron#2015] [Q&A] [answered]
`src\adapter\driver\driver.c 931行 neu_group_split_static_tags(find->group, &find->static_tags, &find->grp.tags);`

上文`find->group = neu_group_new(name, interval);`刚完成初始化，为什么需要立即切分静态点位呢？点位信息是从哪里加载的呢？

**Comment by gc87** [comment, marked as answer]
There are no actual split tags here, just initialized static tags and other tags.

**Reply by MrColinWei** [reply]
明白了，谢谢回复。
